stable/graylog/templates/_helpers.tpl

```diff
@@ -78,7 +78,11 @@ Or use chart dependencies with release name
 */}}
 {{- define "graylog.elasticsearch.hosts" -}}
 {{- if .Values.graylog.elasticsearch.uriSecretKey }}
-    {{- printf "${GRAYLOG_ELASTICSEARCH_HOST}" -}}
+    {{- if .Values.graylog.elasticsearch.uriSSL }}
+        {{- printf "https://${GRAYLOG_ELASTICSEARCH_HOST}" -}}
+    {{- else }}
+        {{- printf "http://${GRAYLOG_ELASTICSEARCH_HOST}" -}}
+    {{- end }}
```

Comment:
Why not just let user add `https` in `.Values.graylog.elasticsearch.uriSecretKey` value?
It's less complicate than having another `uriSSL` prefix.

Reply:
The use case this whole uriSecretKey, uriSecretName and uriSSL features target is the [aws-servicebroker](https://github.com/awslabs/aws-servicebroker). Unfortunately, the broker doesn't provide a way to add `http` or `https` to the secret it generates from the servicebinding.

Reply:
it seems that aws-servicebroker is a kind of service that mess up others